NURESTConnection.j

```diff
@@ -149,18 +149,24 @@ var NURESTConnectionLastActionTimer,
     var responseObject   = [[aConnection responseData] JSONObject],
         responseCode     = [aConnection responseCode],
         containsInfo     = (responseObject && responseObject.errors),
-        errorName,
-        errorDescription;
+        errorMessages         = @[];
+
+    var errorName, errorDescription;
 
     try
     {
-        errorName        = containsInfo ? responseObject.errors[0].descriptions[0].title : nil;
-        errorDescription = containsInfo ? responseObject.errors[0].descriptions[0].description : nil;
+        for (var i = 0; i < responseObject.errors.length; i++)
+        {
+            for (var j = 0; j < responseObject.errors[i].descriptions.length; j++)
```

Comment:
This is the response I get:
```
{
    "errors": [
        {
            "property": "",
            "descriptions": [
                {
                    "title": "Controller-Less Remote Duration Mismatch",
                    "description": "Controller-Less Remote duration should match when using Local-and-Remote on attached profiles. Are you sure you want to proceed ?"
                },
                {
                    "title": "Controller-Less Forwarding Mode Mismatch",
                    "description": "Controller-Less Forwarding Mode should match when using Local-and-Remote on attached profiles. Are you sure you want to proceed ?"
                }
            ]
        }
    ],
    "choices": [
        {
            "id": 1,
            "label": "OK"
        },
        {
            "id": 0,
            "label": "Cancel"
        }
    ]
}
```

`errors` contains an array of objects and each object has `descriptions`, which is an array of `{title, description}`. The second loop is for `descriptions`.